lib/authorizer/hawk.js

```diff
@@ -210,45 +211,61 @@ module.exports = {
                 'extraData',
                 'app',
                 'delegation',
-                'user'
+                'user',
+                'includePayloadHash'
             ]),
             contentType = request.headers.get('content-type'),
 
             // force toString to add a protocol to the URL.
             url = urlEncoder.toNodeUrl(request.url.toString(true)),
 
-            self = this;
+            self = this,
+
+            signRequest = function (bodyHash) {
+                result = self.computeHeader({
```

Comment:
```suggestion
                var result = self.computeHeader({
```
Remove declaration from the parent function.